src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/AddressServiceIntTest.kt

```diff
@@ -83,6 +95,54 @@ class AddressServiceTest : SqsIntegrationTestBase() {
     assertThat(savedAddress?.country).isEqualTo("England")
   }
 
+  @Sql(
```

Comment:
Will update these with tests that test the service layer when it is done. They were useful for now as they found a few bugs in the db script and mapping.